UPGRADE.md

```diff
@@ -621,6 +621,11 @@ following classes and methods:
 
 Use `toIterable()` instead.
 
+## Minor BC BREAK: Changed order of arguments passed to `OneToOne`, `ManyToOne` and `Index` mapping PHP attributes
+
+To keep PHP mapping attributes consistent, order of arguments passed to above attributes has been changed
+so `$targetEntity` is a first argument now. This change affects only non-named arguments usage.
+
```

Comment:
New entries should be added at the top of the file.

Reply:
Pushed to the top. Thanks